node_modules/oae-core/termsandconditions/js/termsandconditions.js

```diff
@@ -21,6 +21,27 @@ define(['jquery', 'oae.core'], function($, oae) {
         var $rootel = $('#' + uid);
 
         /**
+         * Accept the invitation in the URL state, if any
+         */
+        var acceptInvitation = function() {
+            var invitationToken = oae.api.util.url().param('invitationToken');
```

Comment:
That's correct. This is the change the Hilary ticket makes.

https://github.com/oaeproject/Hilary/pull/1238/files#diff-ec09272681a929e971f8efad444b685eR246 sets the invitation token in the create-user process if there is no email provided by external auth.

And, the authentication API already sets the email associated with the invitation token if there is no email set in the create user request (i.e., if you login VIA local auth).